clusterloader2/run-e2e.sh

```diff
@@ -29,14 +29,27 @@ export AZUREDISK_CSI_DRIVER_VERSION="v1.15.0"
 export AZUREDISK_CSI_DRIVER_INSTALL_URL="https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/${AZUREDISK_CSI_DRIVER_VERSION}/deploy/install-driver.sh"
 export WINDOWS_USE_HOST_PROCESS_CONTAINERS=true
 
-# Deploy the GCP PD CSI Driver if required
+# Deploy the GCE PD CSI Driver if required
 if [[ "${DEPLOY_GCI_DRIVER:-false}" == "true" ]]; then
    if [[ -z "${E2E_GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
-      echo "Env var E2E_GOOGLE_APPLICATION_CREDENTIALS must be set to deploy driver"
-      exit 1
+      # Running yq to patch GCE PD CSI Driver manifests so that it runs using
+      # Application Default Credentials instead of creating service account
+      # keys, hence avoiding security risks. See
+      # https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/610
+      # for more details.
+      yq eval -i '
```

Comment:
Do we have any guarantees that there is yq? What if there is not?

Reply:
There is, if one runs CL2 from `kubekins-e2e`: https://github.com/kubernetes/test-infra/blob/ed307a9f6b5fe93c4004e4a2d803fa791c9318c0/images/kubekins-e2e/Dockerfile#L73-L76

Added a precautionary `if` statement with instruction on how to mitigate this.